shared/rocket/build.gradle.kts

```diff
@@ -1,19 +1,54 @@
+import org.jetbrains.kotlin.gradle.plugin.mpp.apple.XCFramework
+
 plugins {
     alias(libs.plugins.quanti.kmp.library)
     alias(libs.plugins.quanti.android.detekt)
     alias(libs.plugins.quanti.android.ktlint)
     alias(libs.plugins.kover)
+    alias(libs.plugins.kotlin.multiplatform)
 }
 
 android {
     namespace = "cz.quanti.rocketrepository"
 }
 
+val libName = "Shared"
+val iosLibFolder = "XCFrameworks/lib"
+
 multiplatformSwiftPackage {
-    packageName("RocketRepository")
+    packageName("Shared")
     swiftToolsVersion("5.3")
     targetPlatforms {
         iOS { v("13") }
     }
     outputDirectory(File(rootDir, "/"))
 }
+
+kotlin {
```

Comment:
And this is the new way? I'd like to see it reusable, i.e. similar to `multiplatformSwiftPackage {}` configuration. Does this make sense? I can try to configure, as it's imo out of scope of your task :)